src/com/ms/silverking/cloud/dht/client/impl/OpSender.java

```diff
@@ -216,15 +217,15 @@ private void createMessages(AsyncOperationImpl[] asyncOpImpls, int startIndex, i
         protoMG = asyncOpImpls[startIndex].createProtoMG(estimate);
         
         if (debug) {
-            System.out.printf("asyncOpImpls.length %d startIndex %d endIndex %d\n", asyncOpImpls.length, startIndex, endIndex);
+            Log.fineAsyncf("asyncOpImpls.length %d startIndex %d endIndex %d", asyncOpImpls.length, startIndex, endIndex);
```

Comment:
debug is off by default. Make these all Log.warningAsyncf so that the debug flag has an effect